index.js

```diff
@@ -21,24 +16,24 @@ function buildTargets(options) {
   return assign({}, defaultTargets, options.additionalTargets);
 }
 
-module.exports = function buildAirbnbPreset(context, options) {
+module.exports = function buildAirbnbPreset(api, options) {
   var transpileTargets = (options && options.targets) || buildTargets(options || {});
 
   var debug = (options && typeof options.debug === 'boolean') ? !!options.debug : false;
 
   return {
     presets: [
-      require('babel-preset-env').default(null, {
+      [require('@babel/preset-env'), {
         debug: debug,
         exclude: [
           'transform-async-to-generator',
-          'transform-es2015-template-literals',
+          'transform-template-literals',
           'transform-regenerator'
         ],
         modules: false,
         targets: transpileTargets
-      }),
-      require('babel-preset-react')
+      }],
+      [require('@babel/preset-react'), { development: api.env('development') }]
```

Comment:
Should this change be documented somewhere?

Reply:
Where is this documented?

Reply:
I edited the PR to mention this, do you want me to add this to the README or somewhere else?

Reply:
If it's going to result in a user-observable difference, it'd be good to document it in the readme.